source/dev_section/programming_tutorials/index.rst

```diff
@@ -21,7 +21,7 @@ Please make sure to setup your environment by following the steps in the *Setup*
        :ref:`installation instructions for developers <dev_installation>`.
 
     #. Download the
-       `EnMAP-Box Workshop 2019 Repository <https://bitbucket.org/hu-geomatics/enmap-box-workshop2019/get/develop.zip>`_
```

Comment:
Better leave as was before. This tutorial is pretty outdated (there is no enmap-box-workshop2019 repository)